homeassistant/components/pvoutput/sensor.py

```diff
@@ -54,13 +55,13 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
     headers = {"X-Pvoutput-Apikey": api_key, "X-Pvoutput-SystemId": system_id}
 
     rest = RestData(method, _ENDPOINT, auth, headers, None, payload, verify_ssl)
-    await rest.async_update()
+    await rest.async_update(hass)
```

Comment:
Why not pass `hass` into the constructor?

Reply:
good idea. done